src/App.js

```diff
@@ -175,7 +171,7 @@ class App extends React.Component {
             this.handleAuthentication(props);
             return <Callback {...props} />
           }}/>
-          <Route path="/login" render={(props) => <Login auth={auth} {...props} />} />
+          <Route path="/login/:itemId/:definitionId" render={(props) => <Login auth={auth} {...props} />} />
```

Comment:
does it make sense to use a queryParam so we can use the same route with path="/login"?

Reply:
Updated